README.md

```diff
@@ -11,6 +11,7 @@ It runs in the same process as the application, so communication overhead is min
 - [Serilog](#serilog)
 - [Microsoft.Extensions.Logging](#microsoftextensionslogging)
 - [ETW (Event Tracing for Windows)](#etw-event-tracing-for-windows)
+- [Application Insights](#application-insights-input)
```

Comment:
Can we make it first? Also please mention all the Application Insights data collectors as a sublist or at least with the reference. Let's start transitioning everybody to use AI as data collector